app/components/model-config.tsx

```
@@ -99,98 +102,102 @@ export function ModelConfigList(props: {
+                title={Locale.Settings.InjectSystemPrompts.Title}
+                subTitle={Locale.Settings.InjectSystemPrompts.SubTitle}
+              >
+                <input
+                  aria-label={Locale.Settings.InjectSystemPrompts.Title}
+                  type="checkbox"
+                  checked={props.modelConfig.enableInjectSystemPrompts}
+                  onChange={e =>
+                    props.updateConfig(
+                      config =>
+                        (config.enableInjectSystemPrompts
+                      = e.currentTarget.checked),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)